site/content/en/topics/assumptions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Assumptions for Provisioning Tools"
+linkTitle: "Assumptions for Provisioning Tools"
+weight: 1
+type: docs
```

Comment:
not sure how this formatting works but is it possible to add a "last modified" or "last revised" date in here?

Reply:
last modified date is generated from commit, hence it is not needed here.